templates/src/pages/Landing/Landing.js

```diff
@@ -3,21 +3,17 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import classnames from 'classnames';
 
-import './Landing.css';
+import './Landing.scss';
 
 import BaseLink from '../../components/BaseLink/BaseLink';
 
+import { default as Transition } from '../PagesTransitionWrapper';
```

Comment:
Maybe just use the custom name without the `{default as }` for "consistency" reasons.
```suggestion
import Transition from '../PagesTransitionWrapper';
```